frontend/pages/create-edit-user.js

```diff
@@ -0,0 +1,424 @@
+                Object.entries(userFormState.links).map(([link, values]) => {
+                    console.log();
+                    if (values.value) {
+                        return [link, values.value];
+                    }
+                })
+            ),
+        };
+        // Deactivate inpute while waiting
+        setTxStatus('WAIT');
+        // Creates a test lens profile with the username as handle
+        if (!lensHandle) {
+            try {
+                const createLensProfile = await lensClient.mutate({
+                    mutation: createProfile(userFormState.username),
+                });
+                if (createLensProfile.data)
+                    setLensHandle(userFormState.username);
+            } catch (err) {
+                alert(
+                    `There was an error creating your lens test profile ${err}`
+                );
+            }
+            resetForm();
+        }
+        try {
+            const ipfontsContract = await connectContract();
+            if (ipfontsContract) {
+                const response = await fetch('./api/user-profile-data', {
+                    method: 'POST',
+                    headers: { 'Content-Type': 'application/json' },
+                    body: JSON.stringify(body),
+                });
+                if (response.status !== 200) {
+                    alert(
+                        'Oops! Something went wrong. Please refresh and try again.'
+                    );
+                } else {
+                    const responseJSON = await response.json(),
+                        cid = `https://${responseJSON.cid}.ipfs.w3s.link/data.json`;
+                    if (!hasIPFonts) {
+                        const txn = await ipfontsContract.createUser(
+                            lensHandle,
+                            cid,
+                            Date.now(),
+                            { gasLimit: 900000 }
+                        );
+                        const wait = await txn.wait();
+                        setTxStatus(!wait && 'SENT');
+                        const reset = setTimeout(resetForm, 2000);
+                        clearTimeout(reset);
+                    } else {
+                        alert('Modify user logic goes here!');
+                        if (lensHandle) {
+                            const getProfile = await lensClient.query({
+                                query: getProfileByHandle,
+                                variables: { handle: lensHandle },
+                            });
+                            console.log(getProfile.data.profile);
+                            const setLensProfile = await lensClient.mutate({
+                                mutation: createSetProfileWithMetadata(
+                                    getProfile.data.profile.id,
+                                    cid
+                                ),
+                            });
```

Comment:
This should be update to ...
```suggestion
                            const setLensProfile = await lensClient.mutate({
                                mutation: createSetProfileWithMetadata,
                                variable({
                                    profileId: getProfile.data.profile.id,
                                    metadata: cid
                                }),
                            });
```
Once `createSetProfileWithMetadata` is updated to accept variables through the mutation

Reply:
Fixed at 4d5af03